docs/user_guide/use_measured_boot.rst

```diff
@@ -87,3 +88,47 @@ An example follows::
 Note: please keep in mind that the IMA-specific options can be combined with
 the above options in the example, resulting in a configuration where a
 `keylime_agent` sent a quote with PCRs [0-15] and both logs (boot and IMA)
+
+Evidently, to be fully used in a meaningful manner, keylime operators need to
+provide it is own custom `mb_refstate` and `mb_policy`. While an user can write
+a policy that performs an "exact match" on a carefully constructed refstate, the
+key idea here is to create a pair of specification files which are at once meaningful
+(for the purposes of trusted computing attestation) and generic (enough to be
+applied to a set of nodes).
+
+The most convenient way to crate an `mb_refstate` is starting from the contents
+of an UEFI boot log from a given node, and then tweak and customize it to make
+more generic. Keylime includes a tool (under `scripts` directory) -
+`generate_mb_refstate` - which will consume a boot log and output a JSON file
```

Comment:
Ah thanks for documenting it, I will try to add it in the distribution package, maybe under keylime-tools.